net/dummy.sh

```diff
@@ -25,12 +25,18 @@ dummy_pre_start()
 	eval dummy="\$type_${IFVAR}"
 	[ "${dummy}" = "dummy" ] || return 0
 
-	ebegin "Creating dummy interface ${IFACE}"
-	if _ip link add name "${IFACE}" type dummy ; then
-		eend 0 && _up && set_interface_type dummy
-	else
-		eend 1
+	if ! modprobe dummy; then
```

Comment:
what if it is built to the kernel?

Reply:
I just tested this, and it seems `modprobe` is fine with attempting to load modules which are actually builtin, it's just a no-op and still returns 0 (which is fine with this logic).

If it is a builtin then the interface would presumably get auto-created at boot time.  Which is also OK thanks to the `_exists` check below.

Reply:
@matoro  how about:
```
if ! test  -d /sys/module/dummy && ! modprobe dummy
```

Reply:
Done and done